libbeat/common/kubernetes/metadata/pod.go

```diff
@@ -89,6 +89,9 @@ func (p *pod) Generate(obj kubernetes.Resource, opts ...FieldOptions) common.Map
 			out.DeepUpdate(meta)
 		}
 	}
+
+	out.Put("pod.ip", po.Status.PodIP)
```

Comment:
This field can be empty in some cases, add it only if is not empty?
```suggestion
	if pod.Status.PodIP != "" {
		out.Put("pod.ip", po.Status.PodIP)
	}
```

Reply:
👍🏼